docs/index.md

```diff
@@ -51,3 +46,17 @@ Additional documentation on the concepts that support the operation of the main
 * [ipc](details/ipc.md): How all the components of Santa communicate.
   duction/syncing-overview.
 * [logs](details/logs.md): What and where Santa logs.
+
+### Binaries
+
+There are five main components that make up Santa, see the following pages to understand their functionality:
+
+* [santad](details/santad.md): A userland root daemon that makes decisions.
```

Comment:
I think you can remove userland from all of these descriptions - it made sense when we had a kernel-land component but now it's redundant